uwsgi.h

```diff
@@ -1515,8 +1522,9 @@ struct wsgi_request {
 	// when set, do not send warnings about bad behaviours
 	int post_warning;
 
-	size_t range_from;
-	size_t range_to;
+	enum uwsgi_range range_parsed;
+	int64_t range_from;
```

Comment:
well i think nowadays being limited to 32bit filesize is not a good thing. Please rename range_from to __range_from and range_to to __range_to and add all the required (64bit) fields to the end of the structure. This will not break binary plugins and will work on 64bit numbers